test/alternator/test_table.py

```diff
@@ -624,29 +624,20 @@ def test_delete_table_description(dynamodb):
     # Seems currently DynamoDB doesn't return CreationDateTime on DeleteTable so we disabled this assertion temporarily.
     # Check when DynamoDB will support it later and re-enable the field's verification.
 
-    # assert 'CreationDateTime' in got
+    assert not 'CreationDateTime' in got
```

Comment:
Please also edit the comment above this assertion to better reflect reality, or just delete the comment entirely.